pallets/runtime/tests/all/settlement_test.rs

```diff
@@ -0,0 +1 @@
+../../../settlement/src/tests.rs
```

Comment:
What changed here?

Reply:
I guess that is how Github display a file move operation when it is a link.